objects/obj_star/Alarm_1.gml

```diff
@@ -502,15 +444,12 @@ if (obj_controller.is_test_map=true){
 // if (obj_controller.is_test_map=true) and (p_owner[2]=1){
 
```

Comment:
i know chimps been trying to get rid of un-needed comments not sure if this applies here?